prow/config/tide.go

```diff
@@ -146,6 +151,9 @@ type Tide struct {
 	//  0 => unlimited batch size
 	// -1 => batch merging disabled :(
 	BatchSizeLimitMap map[string]int `json:"batch_size_limit,omitempty"`
+
+	// Priority
+	Priority []TidePriority `json:"priority,omitempty"`
```

Comment:
If we ever want to add something like this, it needs a lot more documentation than this